ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+**When reporting a bug, make sure you provide all required info below. Reports without this information may be closed.**
+
+
+Output of `docpub --version`:
+
+...
+
+Contents of `docpub.config` file:
```

Comment:
Maybe we might warn users about sensitive fields?
Also, seems everybody care about credentials, so they will not put this info to the public comment. 
In general I would prefer to ask this info because most common issues are usually related to wrong configuration.